tests/python/contrib/test_dnnl.py

```diff
@@ -0,0 +1,206 @@
+'''

```

Comment:
We shouldn't use real network as UT. Please build small unit test for each fusion pattern.